src/Silakov.Homework4/Task.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+
+namespace Silakov.Homework4
+{
+    /// <summary>
+    /// Задача.
+    /// </summary>
+    internal class Task
+    {
+        /// <summary>
+        /// Идентификатор задачи.
+        /// </summary>
+        private int _id;
+        /// <summary>
+        /// Название задачи.
+        /// </summary>
+        private string _name;
+        /// <summary>
+        /// Описание.
+        /// </summary>
+        private string _description;
+        /// <summary>
+        /// Статус задачи.
+        /// </summary>
+        private StatusTask _status;
+        /// <summary>
+        /// Дата создания.
+        /// </summary>
+        private DateTime _creationDate;
+        /// <summary>
+        /// Дата выполнения.
+        /// </summary>
+        private DateTime _executionDate;
+
+        private static int counter = 1;
+
+        Task()
+        {
+            Console.Write("Name: ");
```

Comment:
In general, constructor is not a good place for user input. The purpose of constructor is to instantiate class and initialize fields. Initialization does not mean any interaction, it should just set a correct state for the object. Try to refactor this.